### PR TITLE
WIP: SIG Service Catalog 2020 annual report

### PR DESCRIPTION
**Pull request by jhvhs**
**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by jhvhs** [comment]
cc @jberkhahn @f0rmiga

File: sig-service-catalog/annual-report-2020.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Kubernetes SIG Service Catalog - 2020 Annual report
+Konstantin Semenov, Jonathan Berkhahn
+
+
+[Source](https://github.com/kubernetes/community/blob/master/committee-steering/governance/annual-reports.md)
+
+## Checklist
```
Comment (mrbobbytables) [inline sig-service-catalog/annual-report-2020.md:7]: You can remove the checklist, it's was there to just help you along 👍

File: sig-service-catalog/annual-report-2020.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+       Yes, the [README](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md) is accurate.
+    2. All subprojects correctly mapped and listed in sigs.yaml?
+
+       Yes, our [subprojects](https://github.com/kubernetes/community/blob/master/sig-service-catalog/README.md#subprojects) are current.
+    3. What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date?
+
+       Are you keeping recordings up to date/trends in community members watching recordings?
+
+       We have a low profile bi-weekly meeting, with [notes and agenda up to date](https://docs.google.com/document/d/17xlpkoEbPR5M6P5VDzNx17q6-IPFxKyebEekCGYiIKM/edit#).
+       [Meeting recordings](https://www.youtube.com/watch?v=ukPj1sFFkr0&list=PL69nYSiGNLP2k9ZXx9E1MvRSotFDoHUWs) have not been uploaded since late 2019.
```
Comment (mrbobbytables) [inline sig-service-catalog/annual-report-2020.md:36]: 😬 Do you know what the issue is here? Uploading them is required for transparency reasons. It also helps those that might be interested in your SIG get the latest info on it.

**Comment by mrbobbytables** [comment]
/close

Will revisit for 2021 annual report.

**Pull request closed without merging**